src/TEC/Plugin.php

```diff
@@ -123,17 +126,27 @@ public function register(): void {
 	}
 
 	/**
-	 * Setup our import hooks.
+	 * Run our label hooks.
 	 *
 	 * @since 1.0.0
 	 */
-	public function init_import_hooks() {
+	public function init_label_hooks() {
 		add_filter( 'tec_tickets_commerce_attendee_post_type_args', [ $this, 'tc_attendees_label' ] );
 		add_filter( 'tec_tickets_commerce_order_post_type_args', [ $this, 'tc_orders_label' ] );
 		add_filter( 'tribe_tickets_register_attendee_post_type_args', [ $this, 'rsvp_attendees_label' ] );
 		add_filter( 'tribe_tickets_register_order_post_type_args', [ $this, 'tpp_orders_label' ] );
 		add_filter( 'tec_events_custom_tables_v1_tracked_meta_keys', [ $this, 'modify_tracked_meta_keys' ] );
```

Comment:
```add_filter( 'tec_events_custom_tables_v1_tracked_meta_keys', [ $this, 'modify_tracked_meta_keys' ] );```

Should this hook be moved to the `init_import_hooks` function?

Reply:
Probably yes. I overlooked this.

Reply:
Moved back the meta_keys filter to the import hooks